### ISSUE TITLE
Request: open and close methods

### ISSUE THREAD

**Issue by mafar** [snapappointments/bootstrap-select#1893]
**Objective:**
Similar to ```.selectpicker('toggle')``` , options like close and open are required

1. ```.selectpicker('open')```  open all pickers
1. ```.selectpicker('close')```  close all pickers

**Purpose:**
sometimes it is required to close down all bootstrap-select pickers while ```.selectpicker('toggle')```only switched between open and closed states.
These options will help in achieve either opening or closing one or more selectpickers 

**Solution:**
1. add 2 functions after toggle function in source code
1. Working fiddle after source change http://jsfiddle.net/bababalcksheep/RUAS4/563/

**Tests:**
1. tested with container option as well and works
1. tested and fires all respective events as well

**Sample:**
added ```open: function (e) {``` and ```close: function (e) {``` after toggle in source
```js
    toggle: function (e) {
      e = e || window.event;

      if (e) e.stopPropagation();

      this.$button.trigger('click');
    },

    open: function (e) {
      e = e || window.event;

      if (e) e.stopPropagation();

      if  (!this.$newElement.hasClass('open'))  this.$button.trigger('click');      
    },

    close: function (e) {
      e = e || window.event;

      if (e) e.stopPropagation();

      if  (this.$newElement.hasClass('open'))  this.$button.trigger('click');      
    },
```

**Comment by TomKrakott** [comment]
+1

**Comment by scr4bble** [comment]
What is the status of this issue please?
Is there a way to close the select programmatically?
I only see method for toggle and that's not really what I need. Or is there a way to query whether select is opened so I could filter the opened ones and close them by toggle?

**Comment by caseyjhol** [comment]
This change will be a part of the v1.14.0 release. This issue will be updated after it's made available.

**Comment by caseyjhol** [comment]
Released in [v1.14.0-beta](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.14.0-beta)!
